azure-container-apps-maven-plugin/QUICKSTART.md

```diff
@@ -0,0 +1,70 @@
+# Azure Container Apps Maven Plugin Quick Start
```

Comment:
It seems other plugins do not have this `QUICKSTART.md`. Shall we follow the same practice and remove this file?

We can save the content temporarily, and later write a doc in the documentation site. Then, add the link to the Wiki / Quickstarts.